case-studies/history.md

```diff
@@ -0,0 +1,19 @@
+# History of Tech for Better
+
+We have been doing pro bono work for nonprofits since 2015. Over that time, our approach to that work has been in a continuous process of development.
+
```

Comment:
A little repetitious of the 'pro bono work since 2015'. Not sure if there is a good way to merge para 1 and 2